apps/api/src/scim.ts

```diff
@@ -203,17 +201,15 @@ scimRoute.get('/Users/:userId', passport.authenticate('bearer'), async ( req, re
   
     // Send Response
     res.status(httpStatus).json(userResponse);
-    console.log(userResponse);
   });  
 
   // Update a specific User (PUT)
   // PUT /scim/v2/Users/{userId}
   // RFC Notes on Updating a User: https://www.rfc-editor.org/rfc/rfc7644#section-3.5.1
   scimRoute.put('/Users/:userId', passport.authenticate('bearer'), async (req, res) => {
     // console.log('PUT: /users/:userId'); 
-    console.log(req.body);
+    // console.log(req.body);
```

Comment:
I think we should remove all the commented out console logs unless there's a specific reason to have it tomorrow. This will help streamline the codebase the attendees have to get up to speed on. less cruft is better